client/multi-currency/enabled-currencies-list/style.scss

```diff
@@ -183,5 +189,10 @@
 			min-height: 0;
 			flex: 1;
 		}
+		.enabled-currency-checkbox {
+			.components-checkbox-control__checked {
+				left: 4px;
+			}
+		}
```

Comment:
It works, but maybe we can simplify this by adding just `padding-left: 4px` on `.add-enabled-currencies-modal &__content`.

Reply:
Great suggestion, that's simplifies the code a lot.
Thanks!
Fixed in https://github.com/Automattic/woocommerce-payments/pull/2840/commits/d0f1903dfcd556e40599578b84511d3eea424f8d